task/oracle-cloud-storage-upload/0.1/samples/README.md

```diff
@@ -0,0 +1,107 @@
+# Oracle Cloud Storage Upload - Examples
+
+This directory contains examples demonstrating how to use the `oracle-cloud-storage-upload` task.
+
+## Files
+
+- **`secret-template.yaml`** - Template for creating OCI credentials secret
+- **`taskrun-example.yaml`** - Four TaskRun examples with different configurations:
+  - `oracle-cloud-storage-upload-simple` - Basic upload with parameters
+  - `oracle-cloud-storage-upload-with-secret-config` - Upload using complete secret
+  - `oracle-cloud-storage-sync` - Upload with sync (deleteExtraFiles)
+  - `oracle-cloud-storage-upload-with-extraargs` - Advanced upload with metadata
+- **`pipeline-upload-single-file.yaml`** - Pipeline for single file upload
+  - Pipeline: `upload-to-oci-pipeline`
+  - PipelineRun: `upload-to-oci-pipeline-run-no-prompt`
+- **`pipeline-upload-directory.yaml`** - Pipeline for directory upload
+  - Pipeline: `upload-directory-to-oci-pipeline`
+  - PipelineRun: `upload-directory-to-oci-pipeline-run`
+
+## Quick Start
+
+### 1. Create Credentials Secret
+
+```bash
+# Edit secret-template.yaml with your actual OCI credentials
+kubectl apply -f secret-template.yaml
```

Comment:
NIT: when user browse this on artifacthub they will only see the README. They can of course go to the GitHub repo to see all other files, but perhaps the secret template could also be kept inline in the README.